devenv/dashboards.yaml

```diff
@@ -4,6 +4,6 @@ providers:
  - name: 'gdev dashboards'
    folder: 'gdev dashboards'
    type: file
-   updateIntervalSeconds: 15
+   updateIntervalSeconds: 60
```

Comment:
wont this just be frustrating when working with dashboards?
I think all devs runs SSD nowdays and the content should almost always be cached. scanning a few files are very very very fast :)

Reply:
yea I increased it a bit after testing with many folders defined here (each have it's own interval) and the log spamming was a bit too much. And if we end up with a lot of dashboards parsing all those json files is going to eat up a lot cpu. The way the scanning works provisioning update cycles are far from cheap, having to deserialize all dashboards, and to hashed checksum on content, at least. 

Set to 3s with 1300 dashboards it eats 100% according to this user: https://github.com/grafana/grafana/issues/16118

Reply:
But that 100% could be due to something else, maybe huge > megabytes big json dashboards (can happen)